### ISSUE TITLE
Position of houses seems to be off

### ISSUE THREAD

**Issue by quasar-pankaj** [ygrenier/SwissEphNet#35]
Hello Yan,
I am developing an application using SwissEphNet. I use the following code to get the houses:-
```
using (SwissEph eph = new SwissEph())
			{
				eph.swe_set_ephe_path("Data");
				eph.swe_set_sid_mode(SwissEph.SE_SIDM_LAHIRI, 0, 0);
				var hour = native.Hours + (native.Minutes / 60) + (native.Seconds / 3600);
				var julianNum = eph.swe_julday(year: native.Year, mon: native.Month, mday: native.Day, hour: hour, gregflag: SwissEph.SE_GREG_CAL);
				var builder = new StringBuilder();
				builder.Append(detailsTextBox.Text);

				var cusps = new double[13];
				var ascmc = new double[10];
				var error = eph.swe_houses(tjd_ut: julianNum, geolat: native.Lat, geolon: native.Lng, hsys: 'P', cusps: cusps, ascmc: ascmc);
				foreach (var item in cusps)
				{
					builder.Append(item-eph.swe_get_ayanamsa_ut(julianNum)).Append("\n");
				}
            }
```
After which I input following data:- date of birth 10 jan 1969 at 10:10 am Lat=30.9090157 Lon=75.851601 which gives an ascendant of Gemini which I know to be Pisces. So, this is off by quite a margin. Next I tried using the samples in Swiss ephemeris which also showed similar result. But on the astro site it is correctly reported as Pisces. Am I missing something?

**Comment by ygrenier** [comment]
Hi,

Sorry for the delay, but actually my computer is dead and I'm waiting for a new one, so it's hard to work on my projects.

So did your time zone correction resolve your issue ?

Regards,

**Comment by quasar-pankaj** [comment]
Yes. I was not correcting for timezone. But once I did that everything worked fine.

**Comment by quasar-pankaj** [comment]
Hello Yan,
There is another problem. I wonder if you could guide me here. Actually it is with swe_house_pos() function which is apparently giving wrong results. I am using swe_set_sid_mode() function to select an ayanamsa. and then trying to calculate the house position of the planet. I find the terms used in this function a bit confusing so it is possible that I am not using them correctly.
Right now I am actually comparing the longitude of the planet with that of the house to get the house position. But it would be better if I could use the swe_house_pos() function.
A small sample demo code would be helpful in this regard.
Thanks,
Pankaj
